src/webview/MessageList.js

```diff
@@ -137,15 +140,12 @@ const assetsPath = Platform.OS === 'ios' ? './webview' : 'file:///android_asset/
 // [3] https://github.com/facebook/react-native/blob/0.59-stable/React/Base/RCTConvert.m#L85
 
 class MessageList extends Component<Props> {
-  context: Context;
+  context: ThemeColors;
   webview: ?WebView;
   sendUpdateEventsIsReady: boolean;
   unsentUpdateEvents: WebViewUpdateEvent[] = [];
 
-  static contextTypes = {
-    styles: () => null,
-    theme: () => null,
-  };
+  static contextType = ThemeContext;
```

Comment:
Ah, well spotted finding #1946! I'd forgotten we had an issue on file for that migration. I just added a small comment there with things I learned from a quick look at the history (just now, and also earlier today when we were looking at this.)

Nit here: let's put `static contextType = ...` right above `context: ...`, and then a blank line setting the pair of those off from the other declarations; compare 51dd1b3b2. These two are closely linked. (In fact even with the old API, the old pair of declarations should have been kept together too.)

I want to link to this commit from that issue as an example, so polishing the style matters more than usual :)